tests/test_timeevol.py

```diff
@@ -60,16 +60,37 @@ def local_hamiltonian(t, Omega, phi):
     times = tc.backend.arange(0.0, 3.0, 0.1)
 
     # Evolve with local Hamiltonian acting on qubit 1
-    states = tc.timeevol.ode_evol_local(
+    states0 = tc.timeevol.ode_evol_local(
```

Comment:
for both test functions that use dependence of diffrax, you should first try import diffrax, if error then skip the test. Another way is to add diffrax in the requirements/requirements-extra.txt